testdata/p4_16_bmv_errors/issue679-bmv2.p4

```diff
@@ -29,7 +29,7 @@ parser ParserI(packet_in pk, out H hdr, inout M meta, inout standard_metadata_t
 
 control IngressI(inout H hdr, inout M meta, inout standard_metadata_t smeta) {
 
-    action drop() { smeta.drop = 1; }
+    action drop() { mark_to_drop(); }
```

Comment:
One thing about all these programs called `issueXXX` is that they are testing a particular bug in the compiler. If we edit them we don't know for sure that they are triggering the same bug or not. It is unlikely that the issue is related to the drop field, so probably this is fine. But in general programs called `issue` should not be used as examples.

If you want to remove the drop field from the metadata then you will have to make these edits.

Reply:
I agree there is a danger in changing test programs that they no longer exercise the bug the compiler once had.  If you think it is important before changing these to check that the modified test program gives the same error, with the version of the compiler before the fix, as the original test program, I can do that.

Reply:
Understood.

One more quick thought, if it would be preferable to this PR: I could take the current v1model.p4, make a copy of it with a different name in the testsdata/p4_16_samples directory, e.g. v1model-old1.p4, and have the existing tests that refer to standard_metadata.drop `#include` that one instead of the standard one.  Then the test programs would keep doing what they do now, even if `p4include/v1model.p4` was later changed to remove the `drop` field.